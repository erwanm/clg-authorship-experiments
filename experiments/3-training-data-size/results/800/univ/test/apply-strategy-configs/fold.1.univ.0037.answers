1869-haj-mtmb 1910-rwc-ap	0.307573
1876-cdw-itl 1912-rwc-tsoa	0.053701
1866-espw-gcj 1888-es-ttatv	0.375627
1888-es-ttatv 1907-es-tlotg	0.61866
1880-hj-ws 1904-hj-tgb	0.336398
1893-fmc-md 1900-fmc-itpotk	0.336032
1892-haj-dfg 1903-us-tjoas	0.310772
1885-fmc-tcotk 1893-fmc-tr	0.48876
1866-wta-bos 1882-wta-aa	0.258373
1880-hj-c 1907-ew-tfott	0.514301
1910-ew-tomag 1917-hj-tit	0.502395
1871-haj-tt 1903-haj-bba-ph	0.486539
1882-fmc-mi 1891-fmc-do	0.567578
1888-haj-bb 1893-haj-dtn	0.664475
1875-haj-tyo 1897-haj-fof	0.814574
1868-wta-mob 1883-wta-sh	0.407767
1895-fmc-ajs 1908-fmc-s	0.307261
1883-haj-tbb 1896-hj-toh	0.297347
1900-fmc-itpotk 1916-mt-tms	0.307279
1866-wta-bos 1885-wta-sts	0.172566
1873-hbs-pl 1907-fmc-tdr	0.357157
1887-fmc-pp 1905-haj-fftf	0.588184
1883-fmc-dc 1883-hj-rh	0.395737
1894-cdw-tgh 1910-ab-jwf	0.317101
1874-mt-lotm 1879-hj-h	0.252707
1882-espw-dz 1887-espw-tgb	0.344721
1869-mt-ia 1907-espw-tldup	0.697167
1891-haj-tetb 1912-rwc-tsoa	0.555228
1868-haj-faf 1870-haj-rar	0.612459
1902-fmc-c 1906-fmc-a	0.64274
1914-us-sm 1917-us-kc	0.64655
1863-wta-isao 1886-hj-pc	0.682867
1864-lma-opdaot 1902-fmc-c	0.714834
1899-haj-aoatb 1906-mt-cfma	0.273516
1852-hbs-utc 1862-hbs-tpooi	0.42309
1871-haj-tt 1883-fmc-ars	0.529817
1858-wta-pap 1893-fmc-kl	0.368536
1878-cdw-hikab 1911-us-alp	0.374318
1883-fmc-dc 1884-fmc-aap	0.737837
1886-hj-tb 1901-hj-tsf	0.505161
1881-hj-tpoal 1899-hj-taa	0.537191
1892-fmc-pg 1895-fmc-ajs	0.289138
1874-haj-rftr 1902-mt-adbds	0.360212
1901-hj-tsf 1917-hj-tsotp	0.514297
1881-mt-tpatp 1906-mt-cfma	0.436878
1893-fmc-md 1906-fmc-alor	0.223393
1855-hbs-tmf 1896-hj-tsop	0.514219
1876-haj-btlb 1893-haj-ianwatgfoa	0.64332
1891-fmc-k 1900-rwc-taoamm	0.343836
1897-ew-tdof 1904-rwc-isotu	0.239411
1869-wta-dtr 1881-wta-utr	0.500537
1917-hj-tit 1922-us-tcmc	0.261558
1888-fmc-si 1921-us-tbol	0.420673
1894-cdw-tgh 1897-cdw-tpfwsw	0.529506
1865-wta-waw 1916-ab-tp	0.607265
1872-mt-ri 1906-fmc-alor	0.652707
1866-espw-gsar 1895-espw-tsojc	0.576585
1893-fmc-tr 1896-ab-boat	0.448541
1884-cfw-ea 1895-wta-alae	0.581756
1882-fmc-mi 1903-fmc-thor	0.858392
1897-haj-fof 1906-haj-rotr-ph	0.776635
1879-haj-tym 1891-fmc-k	0.559886
1910-us-sts 1922-us-tcmc	0.459997
1876-haj-btlb 1899-haj-mmv	0.42489
1885-fmc-tcotk 1905-es-tpoe	0.257755
1885-fmc-zamc 1891-fmc-k	0.332585
1885-wta-sts 1895-wta-alae	0.59967
1909-mt-isd 1921-us-tbol	0.340508
1891-fmc-k 1906-fmc-a	0.48953
1891-fmc-k 1893-fmc-kl-v2	0.403489
1869-haj-mtmb 1900-haj-adoh	0.486661
1857-wta-ta 1911-rwc-tcl	0.456879
1885-hj-gr 1902-hj-twotd-v2	0.50644
1868-haj-faf 1883-hj-rh	0.387303
1889-es-tptk 1903-us-tjoas	0.406713
1896-ab-boat 1916-ab-tp	0.295623
1882-haj-ffbts 1893-haj-ianwatgfoa	0.298258
1903-fmc-thor 1916-ew-k-sh	0.398829
1889-espw-tsfi 1915-rwc-b	0.136778
1893-espw-dm 1905-fmc-fm	0.664907
1878-hj-aie 1897-haj-fof	0.270896
1909-rwc-tdm 1915-rwc-p	0.549264
1897-fmc-aroy 1908-fmc-tp	0.280465
1873-wta-tyc 1889-espw-tsfi	0.054901
1884-mt-taohf 1896-ab-boat	0.019787
1867-lma-tagomtt 1868-lma-lw	0.390903
1864-lma-m 1872-lma-ss	0.262073
1887-fmc-s 1900-fmc-itpotk	0.481126
1878-hj-aie 1909-hj-ih	0.201967
1883-cfw-ftm 1902-mt-adbds	0.519272
1902-fmc-c 1908-fmc-s	0.984762
1880-wta-ds 1910-ew-tomag	0.455073
1904-rwc-isotu 1909-rwc-tdm	0.524999
1855-wta-aa 1869-haj-pjb	0.653936
1872-wta-lb 1897-fmc-aroy	0.18751
1869-haj-pjb 1879-haj-ttb	0.659434
1864-lma-m 1919-rwc-tsos	0.246018
1863-wta-isao 1889-wta-wtel	0.483474
1877-hj-ta 1882-fmc-mi	0.35215
1864-wta-tsb 1897-hj-wmk	0.315138
