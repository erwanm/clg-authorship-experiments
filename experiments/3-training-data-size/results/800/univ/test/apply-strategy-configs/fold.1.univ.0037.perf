0.357276	0.626800	0.570000
