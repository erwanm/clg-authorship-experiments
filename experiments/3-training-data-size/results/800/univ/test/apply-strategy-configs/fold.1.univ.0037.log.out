apply-config.sh: applying config '/tmp/expe3/800/process/selected-meta-configs/fold.1.strategy-as-meta.univ.0037.model/strategy-configs/fold.1.univ.0037' to cases '/home/moreaue/dump/expe3//results/800/univ/test/apply-strategy-configs/only-cases.tmp.21Nc91jb'; results written to '/home/moreaue/dump/expe3//results/800/univ/test/apply-strategy-configs/fold.1.univ.0037'
apply-config.sh: done.
